app/controllers/idv/agreement_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
     end
 
     def update
-      skip_to_capture if params[:skip_upload]
+      skip_to_capture if params[:skip_hybrid_handoff] || params[:skip_upload]
```

Comment:
What do you think of changing the new parameter name to `is_mobile` or `mobile_device_detected` or something like that?

Reply:
I think I want to keep it as `skip_hybrid_handoff`. At this moment it makes _slighly_ more sense to me to think of it as the client saying "I don't need to see the hybrid handoff screen" rather than the server saying "you're probably a mobile device with a camera, so I won't show you the hybrid handoff screen".